client/my-sites/stats/activity-log/index.jsx

```diff
@@ -514,7 +514,7 @@ class ActivityLog extends Component {
 			[ 'queued', 'running' ],
 			get( this.props, [ 'restoreProgress', 'status' ] )
 		);
-		const disableBackup = 0 < get( this.props, [ 'backupProgress', 'percent' ], 0 );
+		const disableBackup = 0 <= get( this.props, [ 'backupProgress', 'progress' ], -1 );
```

Comment:
Although `-1` does the job I would love to encourage us all to think about entirely different kinds of types and ranges so as not to introduce confusion when a "fake" value looks real.

One way is with `Infinity`
```js
const disableBackup = 0 <= get( this.props, [ 'backupProgress', 'progress' ], -Infinity )
```

One way that might be clearer is to separate the condition from the value…
```js
const downloadProgress = get( this.props, 'backupProgress.progress' );
const disabledBackup = downloadProgress && downloadProgress <= 0; // disable if a backup is in progress
```

not sure what the appropriate condition is there, but the point is that it's separated from the one type - "percentage of backup progress" - from the other type - "is there a backup in progress"

Reply:
Updated to use `-Infinity`. Thanks Dennis.